portal-web/test/functional/com/liferay/portalweb/macros/FormPortlet.macro

```diff
@@ -59,6 +59,14 @@ definition {
 		FormPortlet.clickSubmit();
 	}
 
+	macro submitAndViewSuccessPage {
```

Comment:
Please delete this macro and use Reanto's submitSuccesfullyAndViewSuccessPage macro in its place